.github/actions/update-snapshots/action.yml

```
@@ -80,7 +80,7 @@ runs:
       continue-on-error: ${{ inputs.continue_on_error == 'yes' }}
       shell: bash -l {0}
       working-directory: ${{ inputs.test_folder }}
-      run: yarn run ${{ inputs.update_script }}
+      run: yarn run ${{ inputs.update_script }} --browser ${{ inputs.browser }}
```

```suggestion
      run: yarn run ${{ inputs.update_script }} --browser '${{ inputs.browser }}'
```

If arguments is empty string on GH Actions side, it will error out:
```
$ playwright test --browser
error: option '--browser <browser>' argument missing
```

If the argument is wrapped in single quotes, it will work and be treated as if it was missing:

```
$ playwright test --browser ''
```

---

Ah nice!